nestkernel/io_manager.h

```diff
@@ -91,26 +92,38 @@ class IOManager : public ManagerInterface
    * SimulationManager::simulate() or SimulationManager::cleanup() by
    * calling the backends' finalize() functions
    */
-  void cleanup();
-  void prepare();
+  void cleanup() override;
+  void prepare() override;
```

Comment:
For consistency the following functions in this class should also be marked `override`:
- `initialize`
- `finalize`
- `change_num_threads`
- `set_status`
- `get_status`

Reply:
done